CMakeLists.txt

```diff
@@ -1209,11 +1209,24 @@ if(BUILD_SHARED)
           COMPONENT PerfTools)
       endif()
     endforeach()
+
+    if(ENABLE_DRAFTS)
```

Comment:
the objects are always built, so you don't need the enable_drafts here or elsewhere